pyproject.toml

```diff
@@ -50,10 +48,6 @@ torch = "*"
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
-[tool.autopep8]
-# autopep8 fights with mypy on line length issue
-ignore = [ "E501" ]
```

Comment:
Please don't remove this. It will still benefit people who have `autopep8` plugin on their IDE.

Reply:
reverted